frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -270,6 +304,25 @@ pub mod pallet {
 			<Self as InstrumentalProtocolStrategy>::rebalance()?;
 			Ok(().into())
 		}
+		/// Occur set access to Account and add it to AdminAccountIds storage.
+		///
+		/// Emits [`AssociatedAccountId`](Event::AssociatedAccountId) event when successful.
+		#[pallet::weight(T::WeightInfo::liquidity_rebalance())]
```

Comment:
this calls `T::WeightInfo::liquidity_rebalance()` by accident

Reply:
Resolved!